openedx/core/lib/api/tests/test_paginators.py

```diff
@@ -107,8 +107,7 @@ def test_paginated_results_last_keyword(self):
         id_range = get_object_range(page_number, page_size)
         db_objects = [build_mock_object(obj_id) for obj_id in id_range]
         self.mock_model.objects.filter = MagicMock(return_value=db_objects)
-
-        page = paginate_search_results(self.mock_model, self.search_results, self.default_size, 'last')
+        page = paginate_search_results(self.mock_model, self.search_results, page_size, 'last')
```

Comment:
@mushtaqak The test suppose to be using page_size less than 6,  `self.default_size` is 6.
Thats why the test was failing. 

As we are also verifying `self.assertFalse(page.has_next())` in test with the result-set of 6 records so I have strong believe the page-size should be less than 6. 
 
@bderusha owns the code, but I think I have explained the reason correctly.